packages/react-native-bpk-component-progress/README.md

```diff
@@ -28,8 +28,8 @@ export default class App extends Component {
   render() {
     return (
       <View style={styles.container}>
-        <BpkProgress min={0} max={100} value={10} />
-        <BpkProgress min={0} max={100} value={10} type="Bar" />
+        <BpkProgress min={0} max={100} value={10} accessibilityLabel="0 of 100" />
+        <BpkProgress min={0} max={100} value={10} accessibilityLabel="0 of 100" type="Bar" />
```

Comment:
`accessibilityLabel` is a required prop so we should include it in our sample code.